lib/shopify_dashboard_plus.rb

```diff
@@ -34,6 +38,25 @@
       exit
     end
   end
+
+  
+  # When adding 44.95.round(2) + 940.6.round(2) the precision of the result will be 985.5500000000001
+  # In a sample of 100_000_000_000 entries, the precision will round up cents
+  # Since all numbers are currency, the plus method will trim to two decimals
+  # Numbers returned as: 44, 44.5, or 44.51
+  class Fixnum
+    def plus(amount)
+      result = self + (amount.to_f rescue 0)
+      result.round(2)
+    end
+  end
+
+  class Float
+    def plus(amount)
+      result = self + (amount.to_f rescue 0)
+      result.round(2)
+    end
+  end
```

Comment:
Using the [money](https://github.com/RubyMoney/money) gem might make dealing with all the rounding stuff easier.